pkg/ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh

```diff
@@ -62,9 +64,13 @@ function docker_network_config() {
 
 DOCKER_NETWORK_OPTIONS='${DOCKER_NETWORK_OPTIONS}'
 EOF
-
-		systemctl daemon-reload
-		systemctl restart docker.service
+		## linux bridge
+		ip link set lbr0 down || true
+		brctl delbr lbr0 || true
+		brctl addbr lbr0
+		ip addr add ${local_subnet_gateway}/${local_subnet_mask_len} dev lbr0
+		ip link set lbr0 up
+		brctl addif lbr0 vlinuxbr
```

Comment:
> @danwinship Perhaps line 73 should be moved back to setup() as that's where vlinuxbr is created

@sdodson yes, since the docker network config update is done before setup(), which is where vlinuxbr is actually created.